test/clj/rems/api/test_catalogue_items.clj

```diff
@@ -82,16 +82,16 @@
   (let [api-key "42"
         owner "owner"
         user "alice"
-        form-id (test-data/create-form! {})
-        wf-id (test-data/create-workflow! {})
-        res-id (test-data/create-resource! {})]
+        form-id (test-data/create-form! {:form/organization "organization1"})
+        wf-id (test-data/create-workflow! {:organization "organization1"})
+        res-id (test-data/create-resource! {:organization "organization1"})]
```

Comment:
I think this is rather ugly that one of the test-data -helpers requires namespaced keys and the others don't. The actual `create-form!` function even recreates the namespaced keys after destructuring.